frontend/src/components/SignUp/SignUp.js

```diff
@@ -15,78 +21,95 @@ class SignUp extends Component {
     }));
   }
 
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  handleSubmit = (event, changeCreatedState) => {
+    event.preventDefault();
+    createUser(this.state)
+      .then((response) => {
+        changeCreatedState(response.status);
+      })
+      .catch((error) => {
+        if (error.message.includes('422')) {
```

Comment:
Is the API returning a 422 if the email already exists? That's odd, I'd expect a 409. But if this is the case, it's cool, and we can revisit it if I can get the API to behave normally.

Reply:
Yeah, it gives a 422. I don't know if that is because of the work around with the confirmation email, or what. It says it's an unprocessable request